cookbooks/lower_crustal_flow/doc/lower_crustal_flow.md

```diff
@@ -0,0 +1,94 @@
+(sec:cookbooks:lower-crustal-flow)=
+# Lower crust flow with an obstacle
+
+*This section was contributed by Cedric Thieulot.*
+
+The setup for this experiment originates in {cite:t}`clark:bush:royden:2005`.
+in which the authors model dynamic stresses associated with the obstruction of lower crustal
+channel flow due to rheological heterogeneity in Eastern Tibet and compare model calculations with
+observed topography where they interpret channel flow of
+the deep crust to be inhibited by the rigid Sichuan Basin as illustrated in {numref}`fig:lower-crust-flow-clark1`
+and {numref}`fig:lower-crust-flow-clark2`
+
+```{figure-md} fig:lower-crust-flow-clark1
+<img src="clark1.*" width="70%" />
+
+3-D perspective of digital topography of eastern and southeastern Tibet, view to the west and northwest.
+Taken from {cite:t}`clark:bush:royden:2005`.
+```
+
+```{figure-md} fig:lower-crust-flow-clark2
+<img src="clark2.*" width="70%" />
+
+General model showing map-view patterns of high and low dynamic pressures. Background flow is from left to right.
+Positive dynamic pressure (i.e. a load acting upwards on the top of the channel) is observed ‘upstream’ of the flow direction and predicts uplift at the surface.
+Taken from {cite:t}`clark:bush:royden:2005`.
+```
+
+A discussion of this model in the context of Eastern tibet is provided in {cite:t}`pitard:replumaz:thieulot:doin:2023`.
+
+The domain is a 3D Cartesian box of size
+$(L_x,L_y,L_z) = (1000~\text{ km}, 500~\text{ km}, 15~\text{ km})$.
```

Comment:
I think the ~ already makes a space so you don;t need the second one; at least in the rendered html version these spaces look really big at the moment. 
```suggestion
$(L_x,L_y,L_z) = (1000~\text{km}, 500~\text{km}, 15~\text{km})$.
```